SWIPs/swip-overlay-address.md

```diff
@@ -0,0 +1,67 @@
+---
+SWIP: <to be assigned>
+title: Swarm node implementer spec - 
+author: Louis Holbrook <dev@holbrook.no> (https://holbrook.no)
+status: Draft
+type: Track Specs
+created: 2019-08-12
+---
+
+## Abstract
+
+This SWIP is a part of a general specification of a Swarm node. The SWIP system is used for convenience only.
+
+## Motivation
+
+Enable other implementations of the swarm node.
+
+## Backwards Compatibility
+
+N/A
+
+## Test Cases
+
+N/A
+
+## Implementation
+
+N/A
+
+## Copyright
+
+Copyright and related rights waived via CC0
+
+## Specification
+
+### Base Hash
+
+Swarm uses `KECCAK256` as its `Base Hash` algorithm. `KECCAK256` has a
+digest size of 32 bytes.
+
+### Swarm Overlay Address
+
+Swarm addresses all content and all nodes using algorithms designed
```

Comment:
Here it seems like it is used for:
- content
- nodes

However, here only node addresses are mentioned. For continuity, it might be good to elaborate on content too (even if if this is just a pointer to a different spec)